### DISCUSSION TITLE
How to add Prism to *.Mobile head while Uno.Ui is required ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#11618] [Q&A]
How to fix this error ? The type "Application" is in ```Uno.Ui``` assembly. But ```Uno.WinUi``` is referenced.

<img width="629" alt="image" src="https://user-images.githubusercontent.com/43340959/223831031-37301cca-8c43-448c-b144-a738ba07588e.png">

<img width="280" alt="image" src="https://user-images.githubusercontent.com/43340959/223831364-2922a661-629c-43f6-90e9-953056d90060.png">

<img width="355" alt="image" src="https://user-images.githubusercontent.com/43340959/223831934-2aac8c15-3c5d-44e2-8576-7ff5fad7f074.png">

Btw, the ```Prism*.dll``` are compiled and are working impeccable with WinUI Desktop App SDK.

**Comment by jeromelaban** [comment]
Thanks. You'll need to provide more information about this issue in a repro sample, screenshots are not enough unfortunately.

**Reply by sigmarsson** [reply]
Thank you for your time.
Please find the rep : https://github.com/sigmarsson/11618

**Reply by jeromelaban** [reply]
I see that you're building prism yourself. You'll need to use the WinUI version of Prism, but it's probably best to wait for the official packages and avoid this kind of fiddling.

**Reply by sigmarsson** [reply]
Spot on, I did. However that home brew assembly works properly with winui desktop.

**Reply by sigmarsson** [reply]
@jeromelaban please advise whether I shall abandon ```Prism``` for Uno Adroid project - if so can you confirm ```NInject``` would work for all heads ? -  or please estimate  when this packge gonna be available so that I can phase out my home brew Prism library. The last official advise from Prism : "You'd want to follow the Uno guidance on creating a WinUI app, and then add Prism.Uno to it." Which is correct in terms of WinUI Desktop application. It works. With that package however the Android head is having the error above.

**Reply by jeromelaban** [reply]
I cannot comment on the work being done on Prism's side, nor provide an estimate. I only know that a newer release is in the works.
